index.d.ts

```diff
@@ -133,6 +136,42 @@ declare namespace camelcaseKeys {
 		*/
 		readonly stopPaths?: readonly string[];
 
+		/**
+		Exclude children at the given object key from being camel-cased.
+
+		If this option can be statically determined, it's recommended to add `as const` to it.
```

Comment:
Use `as const` in the example

Reply:
It would also be useful to document what happens if they don't specify `as const`. What would they miss out on.

Reply:
1/ The example has been updated. 

2/ I agree it would be nice to give a clearer explanation about the use of `as const`. However I'm not sure I'll be able to properly summarize it as I'm not sure to fully grasp all the advantages we get to make the options `readonly` in these cases.

Reply:
But we cannot recommend using `as const` without telling the user why it's recommended. It doesn't have to be extensive.